distribution/linux/flatpak/io.github.openenroth.openenroth.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+DATAFILE="magic7.vid"
+SAVEDIR="$XDG_DATA_HOME/mm7/saves"
+GAMETITLE="Might and Magic VII"
+
+export OPENENROTH_MM6_PATH="$XDG_DATA_HOME/mm6/data"
+export OPENENROTH_MM7_PATH="$XDG_DATA_HOME/mm7/data"
+export OPENENROTH_MM8_PATH="$XDG_DATA_HOME/mm8/data"
+
+if [[ ! $(find "$OPENENROTH_MM7_PATH" -iname $DATAFILE) ]]; then
+    # Create game data/save paths if needed to take at least that step off the user
+    if [[ ! -d "$OPENENROTH_MM7_PATH" ]]; then mkdir -p "$OPENENROTH_MM7_PATH"; fi
+    if [[ ! -d "$SAVEDIR" ]]; then mkdir -p "$SAVEDIR"; fi
```

Comment:
We're using `~/.openenroth` by default for this, can we just use defaults for flatpak?